src/osd/scrubber/scrub_job.h

```diff
@@ -144,6 +145,13 @@ class ScrubJob {
 
   bool high_priority{false};
 
+  /**
+   * If cleared: the scrub can be initiated even if the local OSD has reached
+   * osd_max_scrubs. Only 'false' for those high-priority scrubs that were
+   * operator initiated.
+   */
+  bool observes_max_concurrency{true};
```

Comment:
nit: the name wasn't particularly descriptive; I had to read the comment to grasp the intention.

Reply:
I copied the name from the fuller implementation in 'T', where there are multiple 'observes_*' predicates.